src/util/customTransformer.js

```diff
@@ -78,6 +79,26 @@ async function runUserTransform(
     }),
   );
 
+  await jail.set(
+    '_getGeoLocation',
+    new ivm.Reference(async (resolve, ...args) => {
+      try {
+        const geoStartTime = new Date();
+        let geoData = {};
+        if (GEO_RUDDERSTACK_URL && args.length > 0) {
+          const res = await fetch(`${GEO_RUDDERSTACK_URL}/geoip/${args[0]}`);
```

Comment:
Lets use fetchWithTimeout with timeout set to 99.9p value or the max value of latencies we have seen on geolocation service?

Reply:
done